File: rb/test/exists_test.rb
# RadioSrf1 SDK exists test

require "minitest/autorun"
require_relative "../RadioSrf1_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = RadioSrf1SDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- RadioSrf1 SDK exists test

local sdk = require("radio-srf1_sdk")

describe("RadioSrf1SDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
